docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,664 @@
+Getting started from Scratch
+----------------------------
+
+**Table of Contents**
+
+@@TOC@@
+
+### Intended Audience
+
+This guide is for people who want to craft a custom Kubernetes cluster.  If you
+can find an existing Getting Started Guide that meets your needs on [this
+list](README.md), then we recommend using it, as you will be able to benefit
+from the experience of others.  However, if you have specific IaaS, networking,
+configuration management, or operating system requirements not met by any of
+those guides, then this guide will provide an outline of the steps you need to
+take.  Note that it requires considerably more effort than using one of the
+pre-defined guides.
+
+This guide is also useful for those wanting to understand at a high level some of the
+steps that existing cluster setup scripts are making.
+
+### Prerequisites
+  1. You should be familiar with using Kubernetes already.  We suggest you set
+    up a temporary cluster by following one of the other Getting Started Guides.
+    This will help you become familiar with the tools and API first.
+  1. You should have `kubectl` installed on your desktop.  This will happen as a side
+    effect of completing one of the other Getting Started Guides.
```

Comment:
We do not have instructions for installing kubectl separate from the getting-started-guides.